lib/samples/style_point_with_scene_symbol/style_point_with_scene_symbol.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:flutter/material.dart';
+
+class StylePointWithSceneSymbol extends StatefulWidget {
+  const StylePointWithSceneSymbol({super.key});
+
+  @override
+  State<StylePointWithSceneSymbol> createState() =>
+      _StylePointWithSceneSymbolState();
+}
+
+class _StylePointWithSceneSymbolState extends State<StylePointWithSceneSymbol>
+    with SampleStateSupport {
+  // Create a controller for the scene view.
+  final _sceneViewController = ArcGISSceneView.createController();
+  // A flag for when the scene view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          ArcGISSceneView(
```

Comment:
```suggestion
          // Add a scene view to the widget tree and set a controller.
          ArcGISSceneView(
```
Let's try to catch these things that are the same in every sample.